packages/esm-patient-search-app/src/patient-search-bar/patient-search-bar.component.tsx

```diff
@@ -10,23 +10,27 @@ interface PatientSearchBarProps {
   selectPatientAction?: (patientUuid: string) => void;
   floatingSearchResults?: boolean;
   hidePanel?: () => void;
+  orangeBorder?: boolean;
+  buttonProps?: Object;
 }
 
 const searchTimeout = 300;
 
 const PatientSearchBar: React.FC<PatientSearchBarProps> = ({
   small,
   selectPatientAction,
-  floatingSearchResults,
+  floatingSearchResults = true,
+  orangeBorder,
   hidePanel,
+  buttonProps,
 }) => {
   const { t } = useTranslation();
   const [searchTerm, setSearchTerm] = useState<string>();
   const handleChange = useMemo(() => debounce((searchTerm) => setSearchTerm(searchTerm), searchTimeout), []);
 
   return (
     <div className={styles.patientSearchWrapper}>
-      <div className={styles.searchArea}>
+      <div className={`${styles.searchArea} ${orangeBorder && styles.orangeBorder}`}>
```

Comment:
Probably a better class name would be like active or highlighted instead of orangeBorder.

Reply:
No, it's not when it's active or highlighted, it's only for the global search.